src/cron/schedule-code-review.js

```diff
@@ -0,0 +1,98 @@
+const { CronJob } = require('cron');
+const { ChannelType } = require('discord.js');
+const { translateLanguage } = require('../languages/index');
+const {
+  MAPPED_STATUS_COMMANDS,
+  DISCORD_SERVER,
+  CRON_SHEDULE_REVIEW,
+} = require('../config');
+
+/**
+ * Checks threads for a specific status and sends reminders in all text channels.
+ *
+ * @param {Client} client - The Discord.js client instance.
+ * @param {string} statusText - The status to look for in thread titles.
+ */
+const checkThreadsForReview = async (client, statusText) => {
+  try {
+    const guild = await client.guilds.fetch(DISCORD_SERVER.discordGuildId);
+
+    if (!guild) {
+      console.error(
+        `Guild with ID ${DISCORD_SERVER.discordGuildId} not found.`
+      );
+      return;
+    }
+
+    const channels = await guild.channels.fetch();
+    const textChannels = channels.filter(
+      (channel) => channel.type === ChannelType.GuildText
+    );
+
+    if (textChannels.size === 0) {
+      console.error('No text channels found.');
+      return;
+    }
+
+    for (const channel of textChannels.values()) {
+      try {
+        const threads = await channel.threads.fetchActive();
+        const pendingReviews = threads.threads.filter((thread) =>
+          thread.name.includes(statusText)
+        );
+
+        if (pendingReviews.size > 0) {
+          let messageContent = `🔔 **Pending Threads in ${channel.name}**:\n\n`;
+          for (const thread of pendingReviews.values()) {
+            messageContent += `The thread [${thread.name}] has not been reviewed:\n${thread.url}\n`;
+          }
+
+          await channel.send({
+            content: messageContent,
+          });
+        } else {
+          console.log(
+            translateLanguage('checkReview.noPendingReviews').replace(
+              '{{channelName}}',
+              channel.name
+            )
+          );
+        }
+      } catch (channelError) {
+        console.error(
+          `Error processing channel ${channel.name}: ${channelError.message}`
+        );
+      }
+    }
+  } catch (error) {
+    console.error(
+      `Error fetching guild or processing threads: ${error.message}`
+    );
+  }
+};
+
+/**
+ * Schedules a cron job to run `checkThreadsForReview` at specified intervals.
+ *
+ * @param {Client} client - The Discord.js client instance.
+ *
+ */
+const scheduleReviewCheck = (client) => {
+  const statusText = MAPPED_STATUS_COMMANDS['pr-request-review'];
+  if (!statusText) {
+    console.error('Mapped status for pr-request-review not found.');
+    return;
+  }
```

Comment:
This conditional is two times and in both it uses a plain string. This open possibility that in a future a developer tried to change it and could miss to change it on one of the two sides. Please move this plain string to a variable to make it easy to change in a future or move this validation to a function and then call it in the two sides